src/main/java/com/food/menu/service/UserServiceImpl.java

```diff
@@ -55,4 +60,37 @@ public void updateUser(int userID, User userNew) {
             userOld.setPassword(passwordEncoder.encode(userNew.getPassword()));
         userRepository.save(userOld);
     }
+
+    @Override
+    public List<UserResponse> getUsersSummary(UserSummaryRequest userSummaryRequest) {
+        Session session = sessionFactory.openSession();
```

Comment:
logic needs to be updated here. if the request body is {} then the hql query would be just "FROM User "